conditionals.py

```diff
@@ -33,10 +29,16 @@
    # Write conditional statements that check which is the highest score and prints it.
 
 
+1 = (random.randrange(1, 10))
+2 = (random.randrange(1, 10))
+3 = (random.randrange(1, 10))
```

Comment:
This program does not run currently because these lines are causing a Syntax Error. Instead of `1, 2, 3` here, you should use your variable names. Numbers alone can't be variable names. 

Also, parenthesis aren't necessary around the function calls. 

```suggestion
one = random.randrange(1, 10)
two = random.randrange(1, 10)
three = random.randrange(1, 10)
```

Reply:
done